KK_Pregnancy/GUI/PregnancyGui.StatusIcons.cs

```diff
@@ -138,7 +138,9 @@ private void OnGUI()
                 if (_currentHeroine.Count == 0) return;
 
                 var pos = new Vector2(Input.mousePosition.x, -(Input.mousePosition.y - Screen.height));
-                var heroine = _currentHeroine.FirstOrDefault(x => GetOccupiedScreenRect(x).Contains(pos)).Key;
+                var heroineKeyVal = _currentHeroine.FirstOrDefault(x => GetOccupiedScreenRect(x).Contains(pos));
+                if (heroineKeyVal.Equals(null)) return;
```

Comment:
This will never return true because heroineKeyVal is a struct. You need to check .Value for null instead.
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/
You want to move this check to inside the FirstOrDefault lambda so it becomes 
`var heroine = _currentHeroine.FirstOrDefault(x => x.Value != null && GetOccupiedScreenRect(x).Contains(pos)).Key;`

Reply:
Good catch, it was the x.Value that was null and needed to be caught in GetOccupiedScreenRect